src/ansys/fluent/core/solver/monitors_manager.py

```diff
@@ -111,10 +110,10 @@ def get_monitor_set_data(
             )
 
     def refresh(self, session_id, event_info) -> None:
-        """Monitors refresh callback.
+        """Monitors the refresh callback.
 
         The callback is registered with the EventsManager to refresh plots
-        during initialized and data-read events.
+        during initialization and data-read events.
```

Comment:
I believe he's referring to two event types. I think we can say on-initialized to avoid losing the intended meaning.